test/project_types/script/commands/push_test.rb

```diff
@@ -11,20 +11,28 @@ def setup
         @api_key = "apikey"
         @uuid = "uuid"
         @force = true
-        @env = ShopifyCLI::Resources::EnvFile.new(api_key: @api_key, secret: "shh", extra: { "UUID" => @uuid })
+        @secret = "shh"
+        @env = ShopifyCLI::Resources::EnvFile.new(api_key: @api_key, secret: @secret, extra: { "UUID" => @uuid })
         @script_project_repo = TestHelpers::FakeScriptProjectRepository.new
         @script_project_repo.create(
           language: "assemblyscript",
           extension_point_type: "discount",
           script_name: "script_name",
           env: @env
         )
+        @app = {
+          "title" => "test_app",
+          "apiKey" => @api_key,
+          "apiSecretKeys" => [{ "secret" => @secret }],
+          "appType" => "custom",
+        }
+        Layers::Application::ConnectApp.stubs(:call).returns
```

Comment:
In the two `force_push` tests, I override the stub with the appropriate expect since that's what I am focused on testing there.

This is needed up here to stub out the calls in the other tests, which don't actually care about the result of the ConnectApp request.